backend/apps/ai/common/create_chunks_and_embeddings.py

```
@@ -0,0 +1,56 @@
+                strict=True,
+            )
+            if (
+                chunk := Chunk.update_data(
+                    text=text,
+                    content_object=content_object,
+                    embedding=embedding,
+                    save=False,
+                )
+            )
+        ]
+    except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)